scripts/GenSongsList.py

```diff
@@ -77,6 +61,7 @@ def findMatchingBasename(files, basename):
     # found a song for the first time. Add the title and the filename
     allTitles.append([l(p), str(p)])
 
+downloadExtensions = [".cho", ".chopro", ".mscz"]
```

Comment:
Since there were only two versions of the generated html (download or not), I just created this list of extensions that should get the "download" code. All others will just be displayed in the browser (".url" is a special case treated separately).